dev/dev.clj

```diff
@@ -32,12 +57,38 @@
   extracting data for 2013 for CCGs from specified datasets
   and grouping it by CCG."
   [system]
-  (count (transform/combine-multiple-datasets (:ckan-client system)
-                                              "7a69bc84-fffd-4750-b22b-fc66a5ea0728"
-                                              "0e73fe0d-0b16-4270-9026-f8fd8a75e684")))
+  (count (combine-multiple-datasets (:ckan-client system)
+                                    "7a69bc84-fffd-4750-b22b-fc66a5ea0728"
+                                    "7381b851-7a50-4b8c-b64e-155eadbe5694")))
+
+(defn output-multiple-datasets
+  "Splits dataset by year and outputs data into separate resources."
+  [system field resource_id title]
+  (let [dataset-to-split (storage/get-resource-data (:ckan-client system) resource_id)
+        cleaned-dataset  (transform/remove-ids dataset-to-split)
+        split-resources  (transform/split-by-field-value field cleaned-dataset)
+        new-dataset      (json/encode {:owner_org "kixi"
+                                       :title title
+                                       :name (str "output_muliple_datasets" "_" (quot (System/currentTimeMillis) 1000))
+                                       :author "Kixi"
+                                       :notes "Testing Clojure CKAN client: outputting multiple datasets."})
+        new-package_id   (storage/create-new-dataset (:ckan-client system) new-dataset)]
+    (doseq [[k records] split-resources]
```

Comment:
I'm starting to like reduce for this as a way of keeping a report of the process that can be sent to a log server or kafka or similar. wdyt?